src/custom/state/orders/updater.ts

```diff
@@ -180,15 +180,27 @@ export function EventUpdater(): null {
 
       const block2DateMap = await buildBlock2DateMap(library, logs)
 
-      const ordersBatchData: OrderLogPopupMixData[] = await Promise.all(
-        logs.map(async log => {
-          const { orderUid: id } = decodeTradeEvent(log)
+      // Filter out orders that should not trigger a pop-up
```

Comment:
a bit concern with the updater size, but this is nothing from this PR. It just make it a bit bigger. 
It would be nice to break it down a bit.